lib/os/cbprintf_complete.c

```diff
@@ -1579,11 +1579,15 @@ int cbvprintf(cbprintf_cb out, void *ctx, const char *fp, va_list ap)
 				sign = ' ';
 			}
 
-			if (value->sint < 0) {
-				sign = '-';
-				value->uint = (uint_value_type)-value->sint;
-			} else {
-				value->uint = (uint_value_type)value->sint;
+			{
+				sint_value_type sint = value->sint;
```

Comment:
Any reason this needs to be in a `{ }` block?

Reply:
@galak It's only used within this case statement.  I'd rather limit its scope, instead of making it visible throughout the entire switch body.

If you don't like that, I could move it up to the top of the enclosing `while` loop but that's a pretty wide scope too.

What do you want me to do?

Reply:
enclose the entire switch block in braces if you need to declare variables, or declare them toplevel, anything but this

Reply:
De gustibus non est disputandum.  Done.